recreated PR16235 - Migrate Preact docs from wmr to create-preact (#17226)
cloudflare/cloudflare-docs: src/content/docs/pages/framework-guides/deploy-a-preact-site.mdx

[Preact](https://preactjs.com) is a popular, open-source framework for building modern web applications. Preact can also be used as a lightweight alternative to React because the two share the same API and component model.

In this guide, you will create a new Preact application and deploy it using Cloudflare Pages.
You will use [`create-preact`](https://github.com/preactjs/create-preact), a lightweight project scaffolding tool to set up a new Preact app in seconds.

## Setting up a new project

Create a new project by running the [`npm init`](https://docs.npmjs.com/cli/v6/commands/npm-init) command in your terminal, giving it a title:

```sh
npm init preact
cd your-project-name
```

:::note

> During initialization, select the Prerender app (SSG)? option to allow create-preact to scaffold your app to produce static HTML pages, along with their assets, for production builds. This option is perfect for Pages.

Reviewer: ```suggestion
During initialization, you can accept the `Prerender app (SSG)?` option to have `create-preact` scaffold your app to produce static HTML pages, along with their assets, for production builds. This option is perfect for Pages.
```

> :::note
> 
> **Note:** During initialization, you will notice a "Prerender app (SSG)?" option presented. With this, `create-preact` will scaffold your app to produce static HTML pages - along with their assets - for production builds, which is perfect for Pages.
> 
> :::

Reviewer: ```suggestion
:::note

During initialization, select the Prerender app (SSG)? option to allow create-preact to scaffold your app to produce static HTML pages, along with their assets, for production builds. This option is perfect for Pages.

:::
```

As per @ToriLindsay suggestion (https://github.com/cloudflare/cloudflare-docs/pull/16235#discussion_r1781002152)
